common/environment/setup/sourcepkg.sh

```diff
@@ -4,7 +4,7 @@
 ## VARIABLES
 unset -v pkgname version revision short_desc homepage license maintainer
 unset -v archs distfiles checksum build_style build_helper nocross broken
-unset -v configure_script configure_args wrksrc build_wrksrc create_wrksrc
```

Comment:
this should go in a different commit i think?

Reply:
Maybe, but this commit is very small, so I don't know.